front/src/components/utils/ProfilePicturePlaceholder.tsx

```diff
@@ -0,0 +1,50 @@
+import * as Crypto from 'expo-crypto';
+import { FC, useEffect, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Text, useTheme } from 'react-native-paper';
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    borderRadius: 50,
+    display: 'flex',
+    height: 60,
+    justifyContent: 'center',
+    width: 60,
+  },
+});
+
+type ProfilePicturePlaceholderProps = {
+  username: string;
+};
+
+const ProfilePicturePlaceholder: FC<ProfilePicturePlaceholderProps> = ({
+  username,
+}) => {
+  const [backgroundColor, setBackgroundColor] = useState<string>('#000000'); // Initial color, change as required
+  const theme = useTheme();
+
+  useEffect(() => {
+    const getColor = async () => {
+      const digest = await Crypto.digestStringAsync(
+        Crypto.CryptoDigestAlgorithm.SHA256,
+        username,
+      );
+      let color = '#';
+      for (let j = 0; j < 3; j++) {
+        const value = parseInt(digest.slice(j * 8, j * 8 + 8), 16) % 255;
+        color += ('00' + value.toString(16)).slice(-2);
+      }
+      setBackgroundColor(color);
+    };
+    getColor();
+  }, [username]);
+
+  return (
+    <View style={[styles.container, { backgroundColor: backgroundColor }]}>
+      <Text style={{color: theme.colors.surface}}>{username[0]}</Text>
```

Comment:
Why do you set the text color explicitly?

Reply:
I set it in white because the profile picture generated is always "dark" se if we have a black text we might not see the letter

Reply:
I see. In that case, using `theme.colors.surface` will change the color depending on the current device theme. You should use directly the color palette in `theme.ts` instead. (Export the `palette` object and rename it to `ColorPalette`)

Reply:
done :+1: